aptos-move/framework/src/natives/aggregator_natives/helpers_v2.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+use crate::natives::aggregator_natives::helpers::get_aggregator_field;
+use aptos_aggregator::aggregator_extension::extension_error;
+use move_binary_format::errors::PartialVMResult;
+use move_vm_types::values::{Struct, StructRef, Value};
+
+const VALUE_FIELD_INDEX: usize = 0;
+
+/// Returns ID of aggrgegator snapshot based on a reference to `AggregatorSnapshot` Move struct.
+pub(crate) fn aggregator_snapshot_u128_info(
+    aggregator_snapshot: &StructRef,
+) -> PartialVMResult<u128> {
+    let value = get_aggregator_field(aggregator_snapshot, VALUE_FIELD_INDEX)?.value_as::<u128>()?;
+    Ok(value)
+}
+
+/// Returns ID of aggrgegator snapshot based on a reference to `AggregatorSnapshot` Move struct.
+pub(crate) fn aggregator_snapshot_u64_info(
```

Comment:
why do we call it info? would something like aggregator_snapshot_value_as_u64 be more descriptive?

Reply:
I just used the same naming convention as the existing functions in helpers.rs. I can change the naming convention if you think it's better.

Reply:
Feel free to use a better naming :) I like as_u64 better as well